### PR TITLE
Add `topic list` as an alias for `metadata -t`

### PR DESCRIPTION
**Pull request by SteadBytes**
Closes https://github.com/twmb/kcl/issues/7

Usage examples:
kcl_topic_list

### CODE REVIEW

**Comment by twmb** [comment]
Looks great! I’ll be able to merge on Tuesday once I am back from my mini
vacation!

**Comment by SteadBytes** [comment]
Fantastic - thank you. Have a great vacation! 😄

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+TBD
+===
+
+- `4b9e01f` add `topic list` as an alias for `metadata -t`
```
Comment (twmb) [inline CHANGELOG.md:4]: I think this is missing `(thanks @SteadBytes!)` but I can add that later :)

File: commands/admin/topic/topic.go [hunk 2]
```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/twmb/franz-go/pkg/kmsg"
 
 	"github.com/twmb/kcl/client"
+	"github.com/twmb/kcl/commands/metadata"
```
Comment (twmb) [inline commands/admin/topic/topic.go:19]: Normally I would prefer extracting the logic out to a separate package, but this feels quite clean here, so I think this is great!

**Comment by twmb** [comment]
This is awesome, thank you!

I think I could tag v0.6.0 in a week, I want to think about if there's any updates to franz-go that should happen, and then bump the franz-go dep before tagging.

**Pull request merged**